pkg/validationfile/testdata/caveat_order.yaml

```diff
@@ -0,0 +1,19 @@
+---
+schema: >-
+  definition user {}
+
+  definition resource {
+      relation reader: user with some_caveat
+  }
+
+  caveat some_caveat(somecondition int) {
+    somecondition == 42
+  }
```

Comment:
Was the issue that the caveat was defined below the resource?

Reply:
No, but I figured I'd add a test just the same